src/libs/actions/Policy.js

```diff
@@ -91,6 +91,54 @@ function getPolicyList() {
         });
 }
 
+/**
+ * Remove the passed members from the policy employeeList
+ *
+ * @param {Array} members
+ * @param {String} policyID
+ */
+function removeMembers(members, policyID) {
+
+    // Check if the members array is not empty
+    if (members.length === 0) {
+        return;
+    }
+
+    const key = `${ONYXKEYS.COLLECTION.POLICY}${policyID}`;
+
+    // Make a shallow copy to preserve original data and remove the members
+    const policy = _.clone(allPolicies[key]);
+    policy.employeeList = policy.employeeList.filter(email => !members.includes(email));
+
+    // Optimistically remove the members from the policy
+    Onyx.set(key, policy);
+
+    // Make the API call to merge the login into the policy
+    Policy_Employees_Remove({
+        emailList: members,
+        policyID,
+    })
+        .then((data) => {
+            if (data.jsonCode === 200) {
+                return;
+            }
+            const policyDataWithMembersRemoved = _.clone(allPolicies[key]);
+            policyDataWithMembersRemoved.employeeList = [...policyDataWithMembersRemoved.employeeList, ...members];
+            Onyx.set(key, policyDataWithMembersRemoved);
+
+            // Show the user feedback that the removal failed
+            let errorMessage = translateLocal('workspace.people.genericFailureMessage');
+            if (data.jsonCode === 401) {
+                errorMessage += ` ${translateLocal('workspace.people.onlyAdminCanRemove')}`;
+            }
+            if (data.jsonCode === 402) {
+                errorMessage += ` ${translateLocal('workspace.people.cannotRemovePolicyOwner')}`;
+            }
+
+            Growl.show(errorMessage, CONST.GROWL.ERROR, 5000);
```

Comment:
NAB, do we expect any of these errors to appear for users? Seems like the 401 would happen if an admin could view this page but an admin should not be able to view the page. And we are already preventing from removing the policy owner by filtering them out. 

What we have here is _probably_ fine, but in the future we should make sure we are optimizing for the cases that we expect to encounter today and not the unexpected e.g. instead of catching these specific errors we could have shown just the generic message at the bare minimum or log to the console instead.

Reply:
You are right, technically, we do not expect the two to show up since non-admins should not even be able to see the page and then we decided to filter out the policy owner from the list to remove if they selected themselves. 

I will update it so we only log them to the console and show the generic error message to user.